apps/condo/domains/organization/utils/accessSchema.js

```diff
@@ -49,7 +62,13 @@ async function checkUserBelongsToOrganization (userId, organizationId) {
     return employee.deletedAt === null
 }
 
+const checkIfUserIsOrganizationEmployee = userId => ({ employees_some: { user: { id: userId }, isBlocked: false, deletedAt: null } })
```

Comment:
This function is not an action, that returns true, but is a mapping to subquery.
Different name will help to avoid confusion about usage places (only as subqueries).

```suggestion
const queryOrganizationEmployeeFor = userId => ({ employees_some: { user: { id: userId }, isBlocked: false, deletedAt: null } })
```

Reply:
done